### PR TITLE
Convert t2f models to Python 3-compatible str() methods

### PR DESCRIPTION
**Pull request by caktus-philip**
Convert all models in `t2f` to Python 3-compatible `__str()__` methods using Django's `@python_2_unicode_compatible decorator` while retaining compatibility with Python 2.

https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.python_2_unicode_compatible

Add non-ASCII tests for `str()` and `unicode()` methods for each model.

Only 3 models in `t2f` have `__str__()` methods, so this is a small change.

### CODE REVIEW

File: EquiTrack/t2f/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+@skipIf(sys.version_info.major == 3, "This test can be deleted under Python 3")
+class TestStrUnicode(FastTenantTestCase):
+    '''Ensure calling str() on model instances returns UTF8-encoded text and unicode() returns unicode.'''
+    def test_travel(self):
+        instance = TravelFactory()
+        # This model's __str__() method contains only formatted integers, so it's not possible to challenge it
+        # with non-ASCII text. As long as str() and unicode() succeed, that's all the testing we can do.
+        str(instance)
```
Comment (reinbach) [inline EquiTrack/t2f/tests/test_models.py]: Maybe pass in `reference_number` to `TravelFactory()`  and ensure that that value is part of the `str`/`unicode` result?

Comment (caktus-philip) [reply EquiTrack/t2f/tests/test_models.py]: @reinbach Good suggestion. I thought that `reference_number` was always constructed programmatically (see `make_travel_reference_number()`) but I was wrong. I updated the test.
